spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -213,6 +214,17 @@ public void cleanup(@PathVariable("id") long id) {
 		this.taskDeleteService.cleanupExecution(id);
 	}
 
+	/**
+	 * Stop a set of task executions.
+	 *
+	 * @param ids the ids of the {@link TaskExecution}s to stop
+	 */
+	@RequestMapping(value = "/stop/{id}", method = RequestMethod.POST)
```

Comment:
At second look... we can drop the stop from the request mapping.  This will simplify things further.